### ISSUE TITLE
Strategy to support Symfony 2.x and/or 3.x

### ISSUE THREAD

**Issue by markitosgv** [gesdinet/JWTRefreshTokenBundle#32]
Hi folks,

Recently i've created a new 3.0 branch. Now it works with both Symfony 2.8 and 3.0, but breaks 2.5 - 2.7 support.

Do you think the best option is maintan two branches separately? Or maybe the best option is try to support both in master?

Our principle issue is that interface SimplePreAuthenticatorInterface is moved from Core to Http namespace.

```php
use Symfony\Component\Security\Http\Authentication\SimplePreAuthenticatorInterface;
use Symfony\Component\Security\Core\Authentication\SimplePreAuthenticatorInterface;
```

This interface is used in Gesdinet\JWTRefreshTokenBundle\Security\Authenticator\RefreshTokenAuthenticator

Do you know the best way to "choose" one or another interface depends on Symfony version?

**Comment by dragosprotung** [comment]
I vote for supporting Symfony 2.8 and up, but see #34

**Comment by markitosgv** [comment]
Closed with #35 . Thanks @dragosprotung , i merged and fix styles and travis.yml.

**Comment by dragosprotung** [comment]
Happy to help
